src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,11 +1,12 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private final String bulldozerModel;
```

Comment:
```suggestion
    private final String model;
```
you already know this is a model of bulldozer